karaf/README.md

```diff
@@ -0,0 +1,40 @@
+
+TO run:
+
+1) add these to etc/brooklyn.cfg in the karaf/cloudsoft-amp target assembly (adjusting paths as needed):
+
+brooklyn.experimental.feature.tosca=true
+alien4cloud-config.file=/Users/alex/dev/gits/brooklyn-tosca/brooklyn-tosca-dist/src/main/assembly/files/conf/alien4cloud-config.yml
+
+
+2) whenever karaf is built
+
+feature:repo-add mvn:io.cloudsoft.brooklyn.tosca/brooklyn-tosca-karaf-features/0.10.0-SNAPSHOT/xml/features
+feature:install brooklyn-tosca-karaf-features
+
+
+the feature installation will probably fail; so far, some things have been fixed with:
```

Comment:
Requires updating this README.md to better describe the approach we're taking, and why we're not doing other things.

We are *not* using the `brooklyn-tosca-karaf-features` approach because it will install a huge number of bundles that will have a bunch of import/export packages. That is very high risk for breaking other things running in Karaf by installing different versions of packages that they use. Depending how the other brooklyn bundles (or their upstream dependencies) were written, it would cause us to run with different versions and/or for it to reload bundles, and to make the dependency resolution much harder for Karaf. (for the last point, we already saw OOMEs when Karaf was doing its dependency resolution).

The simplest approach is our low-risk init-jar that contains lots of embedded jars whose packages are not imported or exported.

We then just need a feature that has two bundles: `brooklyn-tosca-karaf-init` and `brooklyn-tosca-karaf-deps-brooklyn`.